builtin/providers/aws/resource_aws_alb_listener_rule.go

```diff
@@ -144,10 +144,15 @@ func resourceAwsAlbListenerRuleRead(d *schema.ResourceData, meta interface{}) er
 	rule := resp.Rules[0]
 
 	d.Set("arn", rule.RuleArn)
-	if priority, err := strconv.Atoi(*rule.Priority); err != nil {
-		return errwrap.Wrapf("Cannot convert rule priority %q to int: {{err}}", err)
+	// Rules are evaluated in priority order, from the lowest value to the highest value. The default rule has the lowest priority.
+	if *rule.Priority == "default" {
+		d.Set("priority", 99999)
```

Comment:
See the comment above, the `default` rule priority is the lowest value, which happens to be `99999` :)